AppCypher/EventsManager
# #153213220 Set up React and Redux

#### What does the PR do?
Create the react-redux architecture for frontend app

#### Description of task to be completed
- Creating react project
- Adding the redux app architecture
- Making the react redux setup work together

#### How should this be manually tested?
- Clone the project repository

  ```$ git clone https://github.com/AppCypher/EventsManager```

- Checkout into the proper branch

  ```$ git checkout ft-react-redux-setup-153213220```

- This project requires Nodejs and it is expected to run in Nodejs version 6 or higher

- Install project dependencies by running

  ```$ npm install```

- There project bundles task runners that make trying out parts of the project easy to use
- Below is a list of some of the available task runners
    - `server` to start the server

    - `test:api` to run the api test suite

    - `build` to start webpack build on all server, frontend and test suite code

    - `build:server` to start webpack build on server code

    - `build:client:dev` to start webpack build on client code under a dev environment variable

    - `build:client:prod` to start webpack build on client code under a production environment variable

- To run the server or test suite, the project code needs to be built

  ```$ npm run build```

#### Relevant pivotal tracker stories
153213220
